c7n/resources/s3.py

```diff
@@ -420,6 +420,7 @@ def handle_BucketWebsiteConfiguration(self, resource, item_value):
      'Notification', None, None, 's3:GetBucketNotification'),
     ('get_bucket_lifecycle_configuration',
      'Lifecycle', None, None, 's3:GetLifecycleConfiguration'),
+    ('get_public_access_block', 'PublicBlock', None, None, 's3:GetBucketPublicAccessBlock')
```

Comment:
@kapilt how do you feel about adding new things to the Augment table? Is there a general consensus on how to proceed? Should I be adding a new value filter instead?

Reply:
general consensus is no more augments, use a dedicated filter.

Reply:
Cool, thanks for confirming. Based off of previous discussions I was beginning to feel that is a potential source of bloat. The action is done. I'll add the tests tonight then start the filter early next week.